src/org/zaproxy/zap/extension/pscanrulesAlpha/StrictTransportSecurityScanner.java

```diff
@@ -41,6 +41,13 @@
 	private static final String MESSAGE_PREFIX = "pscanalpha.stricttransportsecurity.";
 	private static final int PLUGIN_ID = 10035;
 	
+	//max-age=0 disabled HSTS. It's allowed by the spec,
+	//and is used to reset browser's settings for HSTS.
+	//If found raise an INFO alert.
+	private static final String BAD_MAX_AGE="max-age=0";

```

Comment:
Might worth changing to a Pattern to also check the cases where the value is quoted (and with white spaces between the separator).